packages/core/src/generateModel/__tests__/conditional.test.ts

```diff
@@ -559,7 +559,7 @@ describe("conditional", () => {
       }
       function test(): MyConditionalType<number, Hop<number>> { throw new Error() }
     `)
-
+    // console.log(JSON.stringify(modelMap, null, 4))
```

Comment:
Shall we delete this?

Reply:
I have this line in all my tests, and added it to this one. Makes debugging a bit easier 😅